corehq/apps/hqwebapp/urls.py

```diff
@@ -3,8 +3,9 @@
 
 urlpatterns = patterns(
     'corehq.apps.hqwebapp.views',
+    (r'^$', 'redirect_to_default'),
     url(r'^homepage/$', 'redirect_to_default', name='homepage'),
-    url(r'^home/$', 'landing_page', name='landing_page'),
+    url(r'^default_landing/$', 'landing_page', name='landing_page'),
```

Comment:
no one will see this url unless they are running hq locally. Right now `/home` is being hijacked by apache, so no one even sees that.